src/Components/Tasks/Tasks.tsx

```diff
@@ -1,7 +1,130 @@
+  const { baseUrl, requestHeaders }: any = useContext(AuthContext);
+  const { getToastValue }: any = useContext(ToastContext);
+  
+  const [tasks, setTasks] = useState([]);
+  const navigate = useNavigate();
+  // const [itemId, setItemId]: any = useState(0);
+
+
+  // const {
+  //   register,
+  //   setValue,
+  //   handleSubmit,
+  //   formState: { errors },
+  // } = useForm<FormValues>();
+
+  // **********get all tasks*****************
+  const getTasksList = () => {
+    setIsLoading(true);
+    axios
+      .get(`${baseUrl}/Task/manager`, 
+      { headers: requestHeaders })
+      .then((response) => {
+        setIsLoading(false);
+        console.log(response.data);
+        setTasks(response?.data);
+      })
+      .catch((error) => {
+        setIsLoading(false);
+        console.log(error);
+        getToastValue(
+          "error",
+          error?.response?.data?.message ||
+            "An error occurred. Please try again."
+        );
+      });
+  };
+  // **********navigate to add task******************
+  const navigateToNew = () => {
+    navigate("/dashboard/add-task");
+  };
+  useEffect(() => {
+    getTasksList();
+  }, [])
   return (
-    <div>Tasks</div>
+    <>
+    <div className="header d-flex justify-content-between p-3">
+      <h3>Tasks</h3>
+      <button
+        onClick={navigateToNew}
+        className="btn btn-warning rounded-5 px-4"
+      >
+        <i className="fa fa-plus" aria-hidden="true"></i> &nbsp;Add New
+        Task
+      </button>
+    </div>
+    {/* table */}
+    <>
+      <div className="table-container1 vh-100">
+        <table className="table">
+          <thead className="table-head table-bg ">
+            <tr>
+              <th scope="col">Title</th>
+              <th scope="col">Description</th>
+              <th scope="col">Status</th>
+              <th scope="col">User</th>   
+              <th scope="col">Date Created</th>
+            </tr>
+          </thead>
+          <tbody>
+            {tasks?.length > 0 ? (
+              tasks.map((task: any) => (
+                <tr key={task?.id}>
+                  <td>{task?.title}</td>
+                  <td>{task?.description}</td>
+                  <td>Status</td>
+                  <td>Status</td>
+                  <td>
+                    <i
+                      // onClick={() => showViewModal(project?.id)}
+                      className="fa fa-eye  text-info px-2"
+                    ></i>
+                    <i
+                      // onClick={() => showUpdateModal(project)}
+                      className="fa fa-pen  text-warning px-2"
+                    ></i>
+                    <i
+                      // onClick={() => showDeleteModal(project.id)}
+                      className="fa fa-trash  text-danger"
+                    ></i>
+                  </td>
+                </tr>
+              ))
+            ) : (
+              <NoData/>
```

Comment:
This is a div container, and it cannot be a child of the table body, try moving it outside of the table and make it centered in the page

![image](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/baf1b73a-a32c-4098-99cb-cd0978632087)